spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+# rubocop:disable Layout/LineLength
+RSpec.describe 'Showing all posts', type: :feature do
```

Comment:
- [ ] You have some failing tests. Due to this reason I was not able to check your code as requested. Please fix them up :)

> COMMANDS I'm using to test: 
> 1. rails db:reset => to clena up the database
> 2. rspec spec => to test 

<img width="1034" alt="Screenshot 2023-06-01 at 19 36 49" src="https://github.com/mrkamin/Blog-app/assets/69011963/96691bca-e2a5-4867-b844-3f02575cc20f">

Reply:
@Rustamxon7 
thanks for your good and valuable feedback.
now the test passed.
![test passd](https://github.com/mrkamin/Blog-app/assets/108926875/a5e1217e-0c4b-41c5-a2b1-6f26c93d988f)